server/app/forms/admin/BulkStatusUpdateForm.java

```diff
@@ -0,0 +1,42 @@
+package forms.admin;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** A form for selecting multiple applications to be considered for a status update. */
+public final class BulkStatusUpdateForm {
+  private List<String> applicationsIds;
+  private String statusText;
+  private Boolean maybeSendEmail;
```

Comment:
Suggestion (non-blocking): Rename this to `shouldSendEmail`. Java typically uses prefixes like "is" or "should" for boolean variables.